person.rb

```diff
@@ -5,6 +5,11 @@ def initialize(age, name = 'unknown', parent_permission: true)
     @age = age
     @parent_permission = parent_permission
     @corrector = Corrector.new
+    @retals = []
+  end
+
+  def add_rental(date, book)
+    Rental.new(date, book, self)
   end
 
   attr_accessor :name, age
```

Comment:
- [ ] Kindly add the getters and setters at the top of the def initialize method below the class.